docs/administration-guide/data-permissions.md

```diff
@@ -33,19 +45,20 @@ Members of the group can access data from all tables (within all namespaces/sche
 
 Keep in mind people can be in multiple groups. If a person belongs to _another_ group that _does_ have access to that database, that more privileged access will take precedence (overruling the block), and they'll be able to view that question.
 
-## Native query editing
+## Native querying
 
-Members of a group with native query editing set to Yes can write new SQL/native queries using the native query editor. This access level requires the group to additionally have Unrestricted data access for the database in question, since SQL queries can circumvent table-level permissions.
-Members in groups without native query editing access can't view, write, or edit SQL/native queries. People who are not in groups with native query editing permissions will still be able to view the results of questions created from SQL/native queries, but not the code itself. They also won't see the "View the SQL" button when composing custom questions in the notebook editor.
+Members of a group with Native query editing set to Yes can write new SQL/native queries using the [native query editor](https://www.metabase.com/docs/latest/users-guide/writing-sql.html). This access level requires the group to additionally have Unrestricted data access for the database in question, since SQL queries can circumvent table-level permissions.
+
+People who aren't in groups with Native query editing permissions will still be able to view the results of questions created from SQL/native queries, but not the code itself. They also won't see the **View the SQL** button in the query builder.
 
 ## Download results
 
 {% include plans-blockquote.html feature="Download permissions" %}
 
-You can set permissions on whether people in a group can download results (and how many rows) for a data source. Options are:
+You can set permissions on whether people in a group can download results (and how many rows) from a data source. Options are:
 
 - No (they can't download results)
-- Granular (set access for individual tables)
+- Granular (you want to set access for individual tables)
```

Comment:
Maybe it is worth mentioning that it is possible to set granular access to schemas too?